charts/atlantis/Chart.yaml

```
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: v0.27.2
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.24.1
+version: 4.25.1
```

Please set to `4.25.0`. When bumping `minor` it's common to start from `patch` 0.
```suggestion
version: 4.25.0
```

---

Done